beacon_node/beacon_processor/src/work_reprocessing_queue.rs

```diff
@@ -885,18 +861,28 @@ impl<T: BeaconChainTypes> ReprocessQueue<T> {
                     "millis_from_slot_start" => millis_from_slot_start
                 );
 
-                if self
+                match self
```

Comment:
The changes to this block of code are primarily to get rid of the `Clone` on `queued_backfill_batch.clone()`.

The `.clone()` was entirely unnecessary since we actually get the object back when it fails to send down the `ready_work_tx` queue. The clone was significant because it actually clones a `BeaconState` 😱